react/components/MultiSelect/README.md

```diff
@@ -0,0 +1,39 @@
+```js
+initialState = {
+  selected: ["Green", "Red"]
+};
+selectableList = [
+  "White",
+  "Black",
+  "Grey",
+  "Yellow",
+  "Red",
+  "Blue",
+  "Green",
+  "Brown",
+  "Pink",
+  "Orange",
+  "Purple",
+  "Dark-blue",
+  "Dark-red",
+  "Light-blue"
+];
+function sleep(ms) {
+  return new Promise(r => setTimeout(r, ms));
+}
+async function onSearch(term) {
+  await sleep(1000);
```

Comment:
acho que aqui cabe mais um `debounce`, da uma procurada

Reply:
alias me liguei agora que isso ta no readme, nao no componente

eu ainda acho que a busca deveria ser interna, passando uma lista de opçoes no mesmo formato do dropdown (`[{label, value}]`) por padrao, e se o usuario quisesse fazer a propria busca, dai sim ele pode incluir uma prop `onSearch`

Reply:
isso fica como sugestão, se tu discordar pode responder que eu aprovo assim como tá

Reply:
Eu fiz assim pensando no caso onde as opções não cabem na memória e se faz um fetch durante a busca.

Acho que seria possível fazer um "default prop" com essa busca que você sugeriu.

Quanto ao formato `label, value`, eu não sei quanto refatoramento precisaria. Confesso que estou com um pé atrás.

Reply:
isso como default prop seria perfeito

`label, value` sugeri mais por compatiblidade com o dropdown, o que eu acho util (i.e. no momento que tu precisa selecionar mais de um elemento, só muda o componente do styleguide que vai funcionar). 

alem disso acho bom desacoplar o texto visivel do valor real dele

mas se for muita dor trocar, nao precisa

Reply:
(desculpa ser chato quanto a isso, é pq depois pra mudar isso seria um breaking change)

Reply:
Fui olhar sobre `debounce`, e apesar de não ser aqui, acho que valeria a pena adicionar um na função de busca. Ela chama a `prop` (que possivelmente será uma chamada a uma API) a cada caracter mudado. Não é muito performático.

Quero terminar esse PR, mas quero ele redondinho antes disso. Vou tentar adicionar hoje o debounce, o default de busca e o formato compartilhado com o Dropdown. Acho que isso é o mínimo para fechar o código com qualidade.